Jenkinsfile

```diff
@@ -114,14 +114,17 @@ pipeline {
         stage('Unit Test') {
             parallel {
                 stage('run_test.sh') {
-                    agent {
+                    /*agent {
                         /* See if adding dockerfile to test lets it see scons */ 
                         dockerfile {
                             filename 'Dockerfile.centos:7'
                             dir 'utils/docker'
                             label 'docker_runner'
                             additionalBuildArgs '$BUILDARGS'
                         }
+                    }*/
+                    agent {
+                        label 'single'
```

Comment:
Why do you need the _single_ resource?  Is a docker resource not working for some reason?